src/config.rs

```diff
@@ -44,6 +46,30 @@ pub struct ExportersConfig {
     pub influxdb: InfluxDBConfig,
 }
 
+#[derive(Debug, Deserialize, PartialEq)]
+#[serde(rename_all = "lowercase")]
+#[serde(remote = "LevelFilter")]
+pub enum LevelFilterDef {
```

Comment:
You can improve error message like this:
```suggestion
#[serde(rename = "log_level")]
pub enum LevelFilterDef {
```
Error message:
```
Configuration error: enum LevelFilterDef does not have variant constructor asd
```
will be then shown as:
```
Configuration error: enum log_level does not have variant constructor asd
```

Reply:
How about `LoggerConfig log_level` or `LoggerConfig level_filter` (depending on the other change) instead?

Reply:
What do you mean? Are you talking about using some other value in rename attribute?